mechkit/material.py

```diff
@@ -796,17 +795,18 @@ class TransversalIsotropic(AbstractMaterial):
         - **E_l**, **E_t**, **G_lt**, **nu_lt**, **nu_tt**
         - **E_l**, **E_t**, **G_lt**, **nu_tl**, **nu_tt**
 
-    Attributes: **(** Accessible both as attributes and dict-like **)**
+    Attributes:
 
-        - **E_l**, **E_t**, **G_lt**, **G_tt**, **nu_lt**, **nu_tl**, **nu_tt**
+    - Keyword arguments
 
-        - **stiffness** : Stiffness in tensor notation
-        - **stiffness_mandel6** : Stiffness in Mandel6 notation
-        - **stiffness_voigt** : Stiffness in Voigt notation
+    - **stiffness** : Stiffness in tensor notation
```

Comment:
Have you build this locally and looked at the format?
I have not build it locally, but thought that the previous indentation was intended, but I am not sure about this.
Indentation and also level of indentation is usually important in .rst / .md...

Reply:
Updated as discussed.